FinSearchUnified/Tests/Bundle/SearchBundleFindologic/ConditionHandler/CategoryConditionHandlerTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace FinSearchUnified\Tests\Bundle\SearchBundleFindologic\ConditionHandler;
+
+use Exception;
+use FinSearchUnified\Bundle\SearchBundleFindologic\ConditionHandler\CategoryConditionHandler;
+use FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder;
+use Shopware\Bundle\SearchBundle\Condition\CategoryCondition;
+use Shopware\Bundle\StoreFrontBundle\Struct\ProductContextInterface;
+use Shopware\Components\Test\Plugin\TestCase;
+
+class CategoryConditionHandlerTest extends TestCase
+{
+    /**
+     * @var QueryBuilder
+     */
+    private $querybuilder;
+
+    /**
+     * @var ProductContextInterface
+     */
+    private $context = null;
+
+    /**
+     * @throws Exception
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->querybuilder = new QueryBuilder(
+            Shopware()->Container()->get('http_client'),
+            Shopware()->Container()->get('shopware_plugininstaller.plugin_manager'),
+            Shopware()->Config()
+        );
+        $contextService = Shopware()->Container()->get('shopware_storefront.context_service');
+        $this->context = $contextService->getShopContext();
+    }
+
+    public function categoryIdsDataProvider()
+    {
+        return [
+            'Single ID of a category without parents' => [[5], ['Genusswelten']],
+            'One Category without parents and one category having parents' => [
+                [5, 12],
+                ['Genusswelten', 'Genusswelten_Tees und Zubehör_Tees']
+            ],
+            'Category ID of "1"' => [[1], []],
```

Comment:
I actually thought that Shopware would give you an empty string when you try to get the path for "Root". It seems that this is not 100% true.

So here's what are going to do:
* Remove the explicit handling of category with ID 1 from the condition handler.
* Use category "Deutsch" instead (should have the ID 3).

That should accurately represent what we are trying to do.